Just an empty file to create an empty config directory
--- /dev/null
+++ b/.empty
@@ -0,0 +1,1 @@
+
